docs/data/data-grid/row-grouping/row-grouping.md

```diff
@@ -228,41 +239,51 @@ const columns: GridColDef[] = [
 If your column also has a `valueGetter` property, the value passed to the `groupingValueGetter` method will still be the row value from the `row[field]`.
 :::
 
-## Rows with missing groups
+### Rows with missing groups
 
-If the grouping key of a grouping criteria is `null` or `undefined` for a row, the Data Grid will consider that this row does not have a value for this group. and will inline it for those groups.
+If a grouping criterion's key is `null` or `undefined` for a given row, the Data Grid will treat that row as if it doesn't have a value and exclude it from grouping.
+
+The demo below illustrates this behavior—movies are grouped by **Cinematic Universe**, and rows with no value for this column are displayed individually before those with values are displayed in groups:
 
 {{"demo": "RowGroupingRowsWithMissingGroups.js", "bg": "inline", "defaultCodeOpen": false}}
 
 ## Group expansion
 
-By default, all groups are initially displayed collapsed. You can change this behavior by setting the `defaultGroupingExpansionDepth` prop to expand all the groups up to a given depth when loading the data.
-If you want to expand the whole tree, set `defaultGroupingExpansionDepth = -1`
+By default, all groups are initially displayed collapsed.
```

Comment:
I might be wrong but `by default` and `initially` seem to convey a similar meaning.
```suggestion
Initially, all groups are displayed collapsed.
```

Reply:
Unless I've misunderstood, I believe they're conveying two slightly different things: "By default" says this is how the Data Grid is set up to behave out of out the box, whereas "initially" clarifies that this is what happens when data is first loaded (it's displayed collapsed). So it makes sense to use both in this sentence.